governance/handler.go

```diff
@@ -383,6 +384,12 @@ func (gov *Governance) addNewVote(valset istanbul.ValidatorSet, votes []Governan
 				target := gVote.Value.(common.Address)
 				valset.RemoveValidator(target)
 				votes = gov.removeVotesFromRemovedNode(votes, target)
+			case params.Timeout:
+				timeout := gVote.Value.(uint64)
```

Comment:
This type assertion will always succeed?

Reply:
@KimKyungup At this point, yes. Type checking is done before getting into this function.